community.html

```diff
@@ -27,7 +27,12 @@ <h1 class="section-heading">Contact</h1>
     <section>
       <div>
         <h1 class="section-heading">Browse the eco-system</h1>
-        <p class="centered-text">Browse the Eiffel repositories in the <a href="https://github.com/eiffel-community">eiffel-community organization on GitHub</a>.</p>
+        The Eiffel eco-system contains projects that use the Eiffel protocol, both sending and consuming. For details on these projects check them out in the <a href="https://github.com/eiffel-community">eiffel-community organization on GitHub</a>.
```

Comment:
Could we change eco-system to ecosystem on both lines?

Reply:
Fixed in b614d51